content/telegraf/v1.19/introduction/installation.md

```diff
@@ -46,7 +46,14 @@ aren't synchronized with NTP, the timestamps on the data can be inaccurate.
 {{% tab-content %}}
 Debian and Ubuntu users can install the latest stable version of Telegraf using the `apt-get` package manager.
 
-**Ubuntu:** Add the InfluxData repository with the following commands:
+**Ubuntu:** Telegraf is available in recent versions of Ubuntu.
```

Comment:
Make "Ubuntu" and "Debian" headings. There is enough content for each that they need some clear delineation. We should also add headings for installing Telegraf as a package vs building from the repos.